torch/nn/modules/container.pyi.in

```diff
@@ -10,6 +10,7 @@ class Container(Module):
 
 
 T = TypeVar('T')
+ModuleT = TypeVar('ModuleT', bound=Module)
```

Comment:
This is related to #35566. This is actually a pretty clever workaround, maybe we should start applying it everywhere.